docker/presets/models/llama-2/Dockerfile

```diff
@@ -9,7 +9,8 @@ WORKDIR /workspace
 
 # Install git
 RUN apt-get update && \
-    apt-get install -y git && \
+    apt-get upgrade -y && \
```

Comment:
upgrade?

Reply:
that's a necessary step to fix the CVE in base image, e.g. we have been doing this for CSI driver image very long time ago, that could reduce CVEs a lot: https://github.com/kubernetes-sigs/azurefile-csi-driver/blob/a31709bd93c72562030b6afff625e68702a45fb9/pkg/azurefileplugin/Dockerfile#L36